ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js

```diff
@@ -135,9 +140,9 @@ module.exports = Marionette.LayoutView.extend({
         filter,
         isForm: this.options.isForm || false,
         isFormBuilder: this.options.isFormBuilder || false,
+        settingsModel: this.querySettings.currentView.model.attributes.src
```

Comment:
```suggestion
        settingsModel: this.querySettings.currentView.model.attributes.src,
```
brought to you by `/projects/ddf/ui>> yarn format -w`